specs/driver_spec.rb

```diff
@@ -77,4 +77,57 @@
       driver.average_rating.must_equal 0
     end
   end
+
+
+
+
+
+  describe "total_revenue method" do
+    before do
+      @driver = RideShare::Driver.new(id: 3, name: "Lovelace", vin: "12345678912345678")
+      trip = RideShare::Trip.new({id: 8, driver: @driver, passenger: nil, start_time: "2016-08-08", end_time: "2016-08-08T12:16:00+00:00", cost: 17.56, rating: 5})
+      @driver.add_trip(trip)
+
+      # trip = RideShare::Trip.new({id: 8, driver: @driver, passenger: nil, start_time: "2016-08-08", end_time: "2016-08-08T12:16:00+00:00", cost: 5, rating: 3})
+      # @driver.add_trip(trip)
+      #
+      # trip = RideShare::Trip.new({id: 12, driver: @driver, passenger: nil, start_time: "2016-10-08", end_time: "2016-10-08T12:16:00+00:00", cost: 7, rating: 4})
+      # @driver.add_trip(trip)
+      #
+      # trip = RideShare::Trip.new({id: 42, driver: @driver, passenger: nil, start_time: "2016-09-08", end_time: "2016-09-08T12:16:00+00:00",cost: 8, rating: 2})
+      # @driver.add_trip(trip)
+    end
+
+    it "returns total cost of a single drivers trips as a float." do
+      @driver.total_revenue.must_be_kind_of Float
+
+    end
+  end
+
+  describe "avg_hourly_pay" do
+    before do
+      @driver = RideShare::Driver.new(id: 3, name: "Lovelace", vin: "12345678912345678")
+      trip = RideShare::Trip.new({id: 8, driver: @driver, passenger: nil, start_time: "2016-08-08", end_time: "2016-08-08T12:16:00+00:00", cost: 17.56, rating: 5})
+      @driver.add_trip(trip)
+    end
+
+    it "calculates the avg hourly pay for a driver" do
+
+      @driver.avg_hourly_pay.must_be_kind_of Float
+    end
+
+    describe " Driver # driver_status_change" do
+      before do
```

Comment:
This `describe` block is inside the `describe` block for `avg_hourly_pay` - I'm guessing that's not what you intended.